sdk/core/core-amqp/src/errors.ts

```diff
@@ -547,47 +610,62 @@ function isBrowserWebsocketError(err: any): boolean {
   return result;
 }
 
+const rheaPromiseErrors = [
+  // OperationTimeoutError occurs when the service fails to respond within a given timeframe.
+  "OperationTimeoutError",
+
+  // InsufficientCreditError occurs when the number of credits available on Rhea link is insufficient.
+  "InsufficientCreditError",
+
+  // Defines the error that occurs when the Sender fails to send a message.
+  "SendOperationFailedError"
+];
+
 /**
- * Translates the AQMP error received at the protocol layer or a generic Error into a MessagingError.
+ * This utility checks if an error is a retryable error from rhea-promise.
+ * @param err
+ */
+function isRheaPromiseError(err: Error): boolean {
```

Comment:
Nit: Do we really need this utility function? It is used only once. Moreover, retryable errors are determined exactly the same way (calling indexOf on array). Since we dont have a utility for that, but we do for this, it feels out of place.

I would suggest either we remove this utility or add one for retryable errors as well to maintain same patterns in the file.

Reply:
I thought it might help with readability but I'll remove it and inline the indexOf instead.